ElDorito/Source/Patches/ContentItems.cpp

```diff
@@ -363,6 +475,38 @@ namespace
 		return true;
 	}
 
+	wchar_t* GameTypeString(int variantType)
+	{
+		// Perhaps move this into Blam/BlamTypes as GameTypeFileExtension as not all extensions match UI names
```

Comment:
try using a `map` or `unordered_map` to avoid a large switch.

Reply:
I've since moved this into BlamTypes in the following commit
https://github.com/ThiconZ/ElDorito/commit/d58ed6440d1dfd5904ab9a192b452dc5bd566d8d